### ISSUE TITLE
compatibale with python3.9

### ISSUE THREAD

**Issue by epheien** [mil-ad/kitcat#1]
```python
def display_iterm2_new(pixel_data):
    data = b64encode(pixel_data).decode("ascii")

    sys.stdout.write(f"\033]1337;MultipartFile=inline=1;size={len(pixel_data)}\a")
    for chunk in itertools.batched(data, CHUNK_SIZE_IT2):
        sys.stdout.write(f"\033]1337;FilePart={"".join(chunk)}\a")
    sys.stdout.write("\033]1337;FileEnd\a")
    sys.stdout.write("\n")
    sys.stdout.flush()
```
`sys.stdout.write(f"\033]1337;FilePart={"".join(chunk)}\a")` will report syntax error in python3.9

I change to `sys.stdout.write("\033]1337;FilePart={}\a".format("".join(chunk)))` can fix it.

**Comment by mil-ad** [comment]
Thanks for reporting this @epheien! I'll push a fix in a sec

**Comment by mil-ad** [comment]
fixed in #3

Pushed a new release with this fix in.
